shell/actions/order_assign-guestorders2customer.php

```diff
@@ -0,0 +1,85 @@
+<?php 
+
+require_once 'public_html/app/Mage.php';
+umask(0); 
+Mage::init();
+
+
+function getInput($minLength = '1')
+{
+	$handle = fopen ("php://stdin","r");
+	$line = fgets($handle);
+	if(strlen(trim($line)) < $minLength){
+		echo "error - ABORTING!\n";
+		exit;
+	}	
+	fclose($handle);
+
+	return trim($line);
+	
+}
```

Comment:
Clean up empty lines, 4-space tab, spaces:
```suggestion

function getInput($minLength = '1')
{
	$handle = fopen("php://stdin", "r");
	$line = fgets($handle);
	if (strlen(trim($line)) < $minLength) {
            echo "error - ABORTING!\n";
	    exit;
	}	
	fclose($handle);

	return trim($line);
}
```